src/main/java/frc/robot/subsystems/hood/HoodConstants.java

```diff
@@ -17,13 +20,15 @@ public class HoodConstants {
     public static final Measure<Mult<Mult<Mass, Distance>, Distance>> MOMENT_OF_INERTIA =
             Units.Kilograms.mult(Units.Meters).mult(Units.Meters).of(0.0003);
     public static final Translation3d ROOT_POSITION = new Translation3d(-0.27, 0.2385, 0.0);
+    public static final TalonFXConfiguration motorConfiguration = new TalonFXConfiguration();
```

Comment:
Remember to configure current limits, and I also recommend explicitly configuring inversion.

Reply:
10318a4

Reply:
b419731